### PR TITLE
fix(all): update attribution to include libs for consul 1.8.1

### PR DESCRIPTION
**Pull request by jpwhitemn**
Signed-off-by: Jim White <jpwhite_mn@yahoo.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/master/.github/Contributing.md.

## What is the current behavior?
Update 

## Issue Number:
#3163 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## New Imports

- [x] Yes - for Consul 1.8.1 (github.com/hashicorp/go-hclog)
- [ ] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

File: Attribution.txt
Comment (lenny-intel) [inline Attribution.txt:185]: https://github.com/hashicorp/go-hclog isn't in our approved list. But since it is `hashicorp`, I assume we can just add it?

**Review by lenny-intel — CHANGES_REQUESTED** [review]
Need to add the go.mod file updates that triggered these dependencies (i.e. go-mod-registry and go-mod-configuration). The dependaBot PRs will auto close when they see the updates from this PR.

File: Attribution.txt
Comment (jpwhitemn) [reply Attribution.txt:185]: Yes - this will need TSC approval.  Suggesting we approve with "pending TSC approval", then pull it back out and revisit Consul 1.8.1 if not approved.  This in order to get builds going again.

**Comment by cloudxxx8** [comment]
this PR could upgrade go-mod-registry and go-mod-configuration to the latest version in go.mod

github.com/edgexfoundry/go-mod-configuration/v2 v2.0.0-dev.3
github.com/edgexfoundry/go-mod-registry/v2 v2.0.0-dev.3
golang.org/x/crypto v0.0.0-20190923035154-9ee001bba392

**Comment by lenny-intel** [comment]
> this PR could upgrade go-mod-registry and go-mod-configuration to the latest version in go.mod

Yes, this PR needs to also include the go.mod file updates as @cloudxxx8 outlined up.

File: Attribution.txt
Comment (jpwhitemn) [reply Attribution.txt:185]: now approved

**Comment by jpwhitemn** [comment]
Updated to these with latest commit
github.com/edgexfoundry/go-mod-configuration/v2 v2.0.0-dev.3
github.com/edgexfoundry/go-mod-registry/v2 v2.0.0-dev.3
golang.org/x/crypto v0.0.0-20190923035154-9ee001bba392

**Review by lenny-intel — APPROVED** [review]
LGTM

**Pull request merged**
